docs/glossary.md

```diff
@@ -1,8 +1,3 @@
----
-title: 'Glossary'
```

Comment:
I think we need this line back (or push the `# Glossary` header to the absolute first line before the anchors). Otherwise the sidebar renders the title in lowercase.